rllib/core/models/configs.py

```diff
@@ -107,6 +108,51 @@ def build(self, framework: str = "torch") -> Model:
             return TfMLPHead(self)
 
 
+@ExperimentalAPI
+@dataclass
+class CNNEncoderConfig(ModelConfig):
+    """Configuration for a convolutional network.
+
+    See ModelConfig for usage details.
+
+    Attributes:
+        input_dims: The input dimension of the network. It cannot be None.
+        filter_specifiers: A list of lists, where each element of an inner list
+            contains elements of the form
+            `[number of filters, [kernel width, kernel height], stride]` to specify a
```

Comment:
number of filters is also known as  number of channels. might be worth including both.